packages/expo-mail-composer/src/MailComposer.types.ts

```diff
@@ -1,10 +1,34 @@
+/**
+ * A map defining the data to fill the mail.
+ */
 export type MailComposerOptions = {
+  /**
+   * An array of e-mail addresses of the recipients.
+   */
   recipients?: string[];
+  /**
+   * An array of e-mail addresses of the CC recipients.
+   */
   ccRecipients?: string[];
+  /**
+   * An array of e-mail addresses of the BCC recipients.
+   */
   bccRecipients?: string[];
+  /**
+   * Subject of the mail.
```

Comment:
above we say "e-mail addresses" and here we say "mail" - maybe it should be "e-mail" here for consistency? same as for the `body` field